packages/analytics/analytics-chart/src/components/chart-plugins/ChartLegend.vue

```diff
@@ -120,20 +120,20 @@ const position = inject('legendPosition', ref(ChartLegendPosition.Right))
     max-width: 200px;
 
     // Allow legend to expand horizontally at lower resolutions
-    @media (max-width: 768px) {
+    @media (max-width: ($kui-breakpoint-phablet - 1px)) {
```

Comment:
Question: why do you need to do `- 1px` here?

Reply:
Because it's max-width